### PR TITLE
Feature:(Issue 1334) Add support for uint64slices

### LINKED ISSUE #1334: Add `UInt64Slice`
## Checklist

* [x] Are you running the latest v2 release? The list of releases is [here](https://github.com/urfave/cli/releases).
* [x] Did you check the manual for your release? The v2 manual is [here](https://github.com/urfave/cli/blob/master/docs/v2/manual.md)
* [x] Did you perform a search about this feature? Here's the [Github guide](https://help.github.com/en/github/managing-your-work-on-github/using-search-to-filter-issues-and-pull-requests) about searching.

## What problem does this solve?

It seems like `urfave/cli` doesn't currently support Uint64 slices, but supports `Int64Slice` and `UintSlice`. Is there a specific reason why?

## Solution description

Could we incorporate this a `Uint64Slice` flag into a new release?

## Describe alternatives you've considered

Copy-pasting the code for uintslice into my own code... But it would be nice to have this officially supported.

### PR DESCRIPTION
**Pull request by dearchap**
## What type of PR is this?

- feature

## What this PR does / why we need it:

Add support for uint64 slices

## Special notes for your reviewer:

Please focus on altsrc changes as well, especially the type switch

## Testing

Ran unit tests

## Release Notes

```
Add support for uint64slice flag 

```

### CODE REVIEW

**Review by meatballhat — CHANGES_REQUESTED** [review]
Lovely! 🎉 I noticed one bit of what I assume was leftover debugging and I think the new json altsrc stuff should definitely be covered by tests 👍🏼

File: altsrc/flag.go [hunk 1]
```diff
@@ -119,6 +120,47 @@ func (f *IntSliceFlag) ApplyInputSourceValue(cCtx *cli.Context, isc InputSourceC
 	return nil
 }
 
+// ApplyInputSourceValue applies a Int64Slice value if required
+func (f *Int64SliceFlag) ApplyInputSourceValue(cCtx *cli.Context, isc InputSourceContext) error {
+	if f.set != nil && !cCtx.IsSet(f.Name) && !isEnvVarSet(f.EnvVars) && isc.isSet(f.Int64SliceFlag.Name) {
+		value, err := isc.Int64Slice(f.Int64SliceFlag.Name)
+		if err != nil {
+			log.Print(err)
```
[the remark this answers is not in the document]
Comment (dearchap) [reply altsrc/flag.go:128]: yeah will remove

**Comment by meatballhat** [comment]
@dearchap Are you up for addressing the conflicts and getting the test coverage above the threshold? No worries if not. I'm happy to take this work to completion.

**Comment by dearchap** [comment]
@meatballhat Let me take a look

**Comment by dearchap** [comment]
@meatballhat Done. Can you take a look ?

**Comment by dearchap** [comment]
@meatballhat Can you take a look at why tests are failing ? I dont quite understand why.

**Comment by meatballhat** [comment]
@dearchap looks like tests are happy now -- coverage thresholds are still grumpy, but I don't want to be draconian about those checks. Maybe add some coverage threshold wiggle room as part of this PR? e.g.:

**Comment by Dokiys** [comment]
> @meatballhat Can you take a look at why tests are failing ? I dont quite understand why.

codecov will use the coverage from the base commit. Checks show target coverage need 83.22. You can try [this](https://docs.codecov.com/docs/common-recipe-list#set-status-checks-to-block-coverage) specify a target

**Comment by dearchap** [comment]
@meatballhat I've fixed the codecov issues. I'm going to merge.

**Pull request merged**